File: LMS/LMS.Application/DTOs/LibraryManagement/Products/ProductOverviewDto.cs
using LMS.Domain.LibraryManagement.Models.Products;

namespace LMS.Application.DTOs.LibraryManagement.Products
{
    public class ProductOverviewDto
    {
        public Guid ProductId { get; set; }
        public string ProductName { get; set; } = string.Empty;
        public decimal ProductPrice { get; set; }
        public int ProductStock { get; set; }
        public string ImgUrl { get; set; } = string.Empty;
        public decimal DiscountPercentage { get; set; }
        public string ProductDescription { get; set; }
    }
}

File: LMS/LMS.UI/huda-center/src/app/shared/models/library-management/products/product-overview.dto.ts
// src/app/shared/models/library-management/products/product-overview.dto.ts

export interface ProductOverviewDto {
  productId: string;
  productName: string; // هذا سيكون الاسم باللغة المختارة (عربي أو إنجليزي)
  productPrice: number;
  productStock: number;
  imgUrl: string;
  productDescription: string;
  discountPercentage: number; // <--- الخاصية الجديدة
}
